src/style.css

```diff
@@ -1 +1,70 @@
-/* Styles go here */
+* {
+  margin: 0;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  color: #000;
+}
+
+.header {
+  height: 60px;
```

Comment:
do not hardcode heigth, use vertical paddings for linsk

Reply:
I've tried that but 1 px somewhere getting lost and it fails the tests.
I've asked about that in chat and was told to set height of header (https://mate-parttime.slack.com/archives/C03UQS59NRY/p1666945212068099)
And that worked, all tests were passed.
Need your help here.